presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/PlanNodeStatsSummarizer.java

```diff
@@ -118,6 +126,12 @@ private PlanNodeStatsSummarizer() {}
                             (map1, map2) -> mergeMaps(map1, map2, OperatorHashCollisionsStats::merge));
                 }
 
+                // The only statistics we have for Window Functions are very low level, thus displayed only in VERBOSE mode
+                if (operatorStats.getInfo() instanceof WindowOperator.WindowInfo && verbose) {
```

Comment:
Well, the idea was to save merging and collecting unnecessary data if we're not going to display it anyway. Therefore the comment in the code above. If there will be any stats worth displaying for WFs in non-verbose mode, we'd need to rewrite it (remove the `if verbose` bit). If you don't like it, I can rewrite it now and collect stats every time and never display them (for now).